pkg/cache/application_state.go

```diff
@@ -50,13 +50,12 @@ const (
 	KilledApplication
 	ReleaseAppAllocation
 	ReleaseAppAllocationAsk
-	AppStateChange
 	ResumingApplication
 	AppTaskCompleted
 )
 
 func (ae ApplicationEventType) String() string {
-	return [...]string{"SubmitApplication", "RecoverApplication", "AcceptApplication", "TryReserve", "UpdateReservation", "RunApplication", "RejectApplication", "CompleteApplication", "FailApplication", "KillApplication", "KilledApplication", "ReleaseAppAllocation", "ReleaseAppAllocationAsk", "AppStateChange", "ResumingApplication", "AppTaskCompleted"}[ae]
```

Comment:
Note: "RecoverApplication" had to be removed, the string output was actually incorrect.

Reply:
Could you please add unit test to keep consistency between enum and string?

Reply:
Yes, I'll update the PR.